vue/components/molecules/restrictions-alert/restrictions-alert.vue

```diff
@@ -67,28 +83,63 @@ body.mobile .restrictions-alert .message-restrictions-alert .button-container {
     text-decoration: underline;
     user-select: none;
 }
+
+.restrictions-alert .message-restrictions-alert-container .message-restrictions-alert .message-container {
+    max-height: 200px;
+    overflow-y: auto;
+}
+
+.restrictions-alert .message-restrictions-alert-container .message-restrictions-alert .message-container .messages .name,
+.restrictions-alert .message-restrictions-alert-container .message-restrictions-alert .message-container .messages .value {
+    display: inline-block;
+}
+
+.restrictions-alert .message-restrictions-alert-container .message-restrictions-alert .message-container .messages .name {
+    font-weight: 600;
+    padding: 0px 2px 0px 0px;
+    text-transform: capitalize;
+}
 </style>
 
 <script>
 export const RestrictionsAlert = {
     name: "restrictions-alert",
     data: function() {
         return {
-            visible: false
+            visible: false,

```

Comment:
The way this is being mutated is getting confusing.

Let's try changing this to be a `computed` based on the `messages` and `changes` arrays. Then, clear those arrays when there is a "change" event (something like: `config`, `part`, `initials`, `initials_extra`).